.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,58 +1,25 @@
 _Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_
 
-### Related issues
-
-<!-- Please link any related issues here. -->
-
-### Description
-
-<!-- Describe your changes in detail -->
-
-### Motivation & Context
-
-<!-- Why is this change required? What problem does it solve? -->
-
 ### Types of change
 
-<!-- What types of changes do your code introduce? -->
-<!-- Please remove the unused items in the list -->
-
-- Bug fix (non-breaking which fixes an issue)
-- New feature (non-breaking change which adds functionality)
-- Refactoring (non-breaking change)
-- Breaking change (fix or feature that would change existing functionality)
-
-### Previews
+- [ ] Non-breaking change
+    - [ ] Bug fix
+    - [ ] New feature
+    - [ ] Refactoring (no functional changes, no api changes)
+    - [ ] Other (documentation, CI/CD, build scripts...)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 
-<!-- Please add screenshots or videos showing your evolutions -->
-
-### Checklist
-
-<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
-<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-<!-- Note that any checkboxes not optional must be ticked by an 'x' (or '(NA)') and our [branch ruleset](https://github.com/marketplace/task-list-completed) may block any merge if some mandatory boxes remain empty -->
-<!-- Your branch used to submit the evolutions must be prefixed by the issue number like 666-add-some-feature -->
-
-#### Contribution
-
-- [ ] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/ouds-android/blob/develop/CONTRIBUTING.md)
-
-#### Accessibility
-
-- [ ] My change follows accessibility good practices
-
-#### Design
+### Related issues
 
-- [ ] My change respects the design guidelines of _Orange Unified Design System_
+Closes #
 
-#### Development
+### Motivation & Context
 
-- [ ] My change follows the [developer guide](https://github.com/Orange-OpenSource/ouds-android/blob/develop/DEVELOP.md)
-- [ ] I have added unit tests to cover my changes _(optional)_
+<!-- Why is this change required? Describe the actual behavior (and how to reproduce it). -->
 
-#### Documentation
+### Description
 
-- [ ] My change introduces changes to the documentation and/or I have updated the documentation accordingly
+<!-- Describe your changes in detail -->
 
 ### Checklist (for Core Team only)
```

Comment:
The last item in the checklist could be removed because is it now automatic:

```
changelog.md has been updated respecting [keep a changelog rules](https://keepachangelog.com/en/1.0.0/) and referencing the issue
```